### PR TITLE
Edited titles of topic pages

### PR DESCRIPTION
**Pull request by bharmalh**
Removed the "- web://workbook" part from all the index.md files within the topics folder as it is already being appended during site generation.

### CODE REVIEW

File: content/topics/protocols/index.md [hunk 1]
```diff
@@ -75,13 +75,13 @@ In other words, it takes the data from the application layer, splits into smalle
...
+  - [TCP](https://developer.mozilla.org/en-US/docs/Glossary/TCP)
+  - [Packet](https://developer.mozilla.org/en-US/docs/Glossary/Packet)
+- Wikipedia related entries
+  - [TCP](https://pt.wikipedia.org/wiki/Protocolo_de_controle_de_transmiss%C3%A3o)
 
 ### SMTP
 
 > SMTP stands for Simple Mail Transfer Protocol - This is an email protocol used for sending emails from one email account to another via the internet. SMTP server handles the sending, receiving, and relaying of email. You can think of servers as your real-life post offices. When you send a letter from city A to city B, it first reaches a local post office in city A; here, it gets processed and sent to the post office in city B, which is in charge of delivering it to its final destination. The same happens with SMTP servers—though instead of taking days, the process takes a few minutes at most.
-*[Read More Here](https://postmarkapp.com/guides/everything-you-need-to-know-about-smtp)*
+> _[Read More Here](https://postmarkapp.com/guides/everything-you-need-to-know-about-smtp)_
```
Comment (schalkneethling) [inline content/topics/protocols/index.md:87]: Was this intentional? It changes it from a bulleted list to an emphasized block quote.

**Review by schalkneethling — CHANGES_REQUESTED** [review]
Thanks, @bharmalh - Just one item to address.

File: content/topics/protocols/index.md (hunk 1 above)
Comment (bharmalh) [reply content/topics/protocols/index.md:87]: No, this was not intentional. Sorry about that.

Comment (schalkneethling) [inline content/topics/protocols/index.md:87]:
```suggestion
- [Read More about SMTP](https://postmarkapp.com/guides/everything-you-need-to-know-about-smtp)
```

Comment (schalkneethling) [reply content/topics/protocols/index.md]: No worries, I have reverted the change.

**Review by schalkneethling — APPROVED** [review]
Thank you for your contribution, @bharmalh 🎉

**Pull request merged**